client/src/components/Tea/TeaList/TableColumnHeader.jsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faAngleDown, faAngleUp } from "@fortawesome/free-solid-svg-icons";
+
+library.add(faAngleDown, faAngleUp);
+
+export class TableColumnHeader extends React.Component {
+  sortColumn = () => {
+    return (
+      this.props.sortColumn === this.props.columnName &&
+      this.props.sortOrder === "asc"
+    );
+  };
+
+  render() {
+    const columnName =
+      this.props.columnName === "teaType"
```

Comment:
This doesn't buy you a ton of flexibility with your column names. I don't think it's a "must fix" for this PR, but if I were you, I'd decouple the attribute name and the display name.  Just pass them in as separate props or have an object here that maps the attr name to its display name.

Reply:
Agreed - please map to display names. This technique is brittle especially when considering internationalization or adding new columns with multiple words.

Reply:
Agreed. I was trying to find some efficient way to do this. Separate props sounds fine with me.

Reply:
Mapping will probably be easier for devs to follow.  You have a single object that contains all the attr-columnName pairs and can see it all in a glance, instead of needing to pass more props into your TableColumnHeader. But up to you!